loadtest/store/memstore/store.go

```diff
@@ -315,3 +316,42 @@ func (s *MemStore) Roles() ([]model.Role, error) {
 	}
 	return roles, nil
 }
+
+// For all the random utilities we flatten the map to a slice
```

Comment:
I would put everything related to random picking data in a separate file since it's very likely this will grow further.
Also I would implement functions to randomly pick:

- A post
- An emoji
- A channel member
- A team member

This way we cover pretty much everything we currently store.

Reply:
Ah okay, then it might be better to use a util function with `interface{}`.

Reply:
Yes, that's probably a good option and as long as we keep it internal to the package we shouldn't have problems of being too generic.